website/src/content/api-reference/deployment-previews.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you test your Custom Application features before publishing them to the production environment.
+
+</Subtitle>
+
+This document describes the necessary steps to build and add deployment previews to your Custom Application in the Merchant Center.
+
+The process involves:
+
+1. Building deployment previews for your Custom Application.
+2. Deploying the Custom Application preview to your hosting provider.
+3. [Adding the deployment preview](https://docs.commercetools.com/merchant-center/managing-custom-applications#adding-deployment-previews) to your Custom Application settings.
+
+# Building deployment previews
+
+Each deployment preview has its unique and stable deployment URL. To build a deployment preview, you must set the `env.production.url` value in the `custom-application-config.json` file to the deployed application. You can follow one of the below approaches:
+
+## Using environment variables
+
+To configure your application URL, you can use environment variables. Then, you can assign different values to the specified environment variable in each deployment environment. The Custom Application `build` script will automatically resolve the application URL from the environment variables.
```

Comment:
nit: maybe [`mc-scripts build`](https://docs.commercetools.com/custom-applications/api-reference/cli#build), not the Custom Application `build` script

Reply:
Thanks. Good suggestion.